### PR TITLE
Reorganizing "Adding registries in Che" content

### PR DESCRIPTION
**Pull request by sishida**
Signed-off-by: Sarah Ishida <sishida@us.ibm.com>

Questions about this PR:
- Line 15: Should “IBM Cloud Private” be replaced with another name?
- Which "Next Step" link do we want? I'm thinking maybe `[Adding the OpenShift internal registry with Codewind](openshiftregistry.html)` might be the way to go?

### CODE REVIEW

File: docs/_documentations/mdt-che-setupregistries.md [hunk 1]
```diff
@@ -1,25 +1,80 @@
...
+These examples show recommended values for common registries. The following deployment registries are tested and verified with Codewind:
+- Docker Hub:
+    - Address: `docker.io`
+    - Namespace: `<namespace>`
+    - Credentials: Docker Hub user name and password or access token
+    - **Note:** For Docker Hub, the `Namespace` value is likely to be your user name. 
+- Quay.io:
+    - Address: `https://quay.io`
+    - Namespace: `<namespace>`
+    - Credentials: Quay.io user name and encrpyted password
```
Comment (tetchel) [inline docs/_documentations/mdt-che-setupregistries.md]: Typo encrpyted

File: docs/_documentations/mdt-che-setupregistries.md [hunk 2]
```diff
@@ -1,25 +1,80 @@
...
-### Adding registries in Che
-After Che is started and running, add the container registry that will be used with Codewind.
-- On OpenShift or other Kube platforms, you can push your images to any Docker registry, such as Docker Hub, Quay.io, Google Cloud Registry (GCR), and more.
-- On IBM Cloud Private, you can push your image to any Docker registry *except* the internal Docker registry.
+# Adding registries in Che
+After Che is started and running, add the container registry to be used with Codewind.
 
-You'll need the following information to complete the instructions to add the registries:
+- On OpenShift and other Kube platforms, push your images to any Docker registry, such as Docker Hub, Quay.io, Google Cloud Registry (GCR), and more.
+- On IBM Cloud Private, you can push your image to any Docker registry except the internal Docker registry.
```
Comment (sishida) [inline docs/_documentations/mdt-che-setupregistries.md]: Should “IBM Cloud Private” be replaced with another name?

File: docs/_documentations/mdt-che-setupregistries.md [hunk 3]
```diff
@@ -1,25 +1,80 @@
...
+    - Run Codewind locally to develop an Appsody style project.
+    - The image registry for the Appsody stack requires credentials. 
+
+If you do not develop any Codewind style projects, and you use an image registry that does not require credentials, then you do not need to specify an image registry.
+
+### Adding registries to OKD and OpenShift
+To use the OpenShift internal Docker registry with Codewind, see [Adding the OpenShift internal registry with Codewind](openshiftregistry.html).
 
 Next step: [Create Codewind workspace](mdt-che-createcodewindworkspace.html)
+Next step: [Using Codewind in Theia](mdt-che-usingtheia.html)
```
Comment (sishida) [inline docs/_documentations/mdt-che-setupregistries.md]: Which "Next Step" link do we want? I'm thinking maybe [Adding the OpenShift internal registry with Codewind](openshiftregistry.html) might be the way to go?

File: docs/_documentations/mdt-che-setupregistries.md (hunk 2 above)
Comment (johnmcollier) [reply docs/_documentations/mdt-che-setupregistries.md]: We can remove the IBM Cloud Private section altogether.

File: docs/_documentations/mdt-che-setupregistries.md (hunk 3 above)
Comment (johnmcollier) [reply docs/_documentations/mdt-che-setupregistries.md]: I don't know if I would put it as the next step here. It's kind of like a sibling doc to this one.

File: docs/_documentations/mdt-che-setupregistries.md [hunk 4]
```diff
@@ -1,25 +1,75 @@
...
 type: document
-order: 20
-parent: root
 ---
 
-### Adding registries in Che
-After Che is started and running, add the container registry that will be used with Codewind.
-- On OpenShift or other Kube platforms, you can push your images to any Docker registry, such as Docker Hub, Quay.io, Google Cloud Registry (GCR), and more.
-- On IBM Cloud Private, you can push your image to any Docker registry *except* the internal Docker registry.
+# Adding registries in Che
```
Comment (johnmcollier) [inline docs/_documentations/mdt-che-setupregistries.md]: I would change the title to something like: `Adding a Docker registry in Codewind`

File: docs/_documentations/mdt-che-setupregistries.md [hunk 5]
```diff
@@ -1,25 +1,75 @@
...
+After Che is started and running, add the container registry to be used with Codewind.
+1. From the Codewind Project Explorer view, right-click **Projects** and select **Image Registry Manager**.
+2. From the **Image Registry Manager**, click **Add New**.
+3. On OpenShift and other Kubernetes platforms, push your images to a container registry, such as Docker Hub, Quay.io, Google Cloud Registry (GCR), and more. Enter the following information to add the registries:
```
Comment (johnmcollier) [inline docs/_documentations/mdt-che-setupregistries.md]: Change this to something like: `Enter the following information to add the registries`

File: docs/_documentations/mdt-che-setupregistries.md [hunk 6]
```diff
@@ -1,25 +1,75 @@
...
+  - Registry server name or domain name: `<registry-to-push-images-to>`
+  - User name: `<Your username>`
+  - Password or API key: `<Your password or API key>`
+4. Next, enter the namespace that you want to use to push your registry to. After you enter all the information, from the **Image Registry Manager**, you can see values for the **Address**, **Username**, and **Namespace** of your latest registry.
```
Comment (johnmcollier) [inline docs/_documentations/mdt-che-setupregistries.md]: Should be something like: `If you're using that registry as a push registry, enter the namespace that you wish to push your images to`

I think we should also add a note somewhere about the examples we have down below.

File: docs/_documentations/mdt-che-setupregistries.md [hunk 7]
```diff
@@ -1,25 +1,75 @@
...
+5. Click **Select a Push Registry** to determine which registry you use to push Codewind style projects to.
 
-You'll need the following information to complete the instructions to add the registries:
-  - Server: `<registry-to-push-images-to>`
-  - Username: `<Your username>`
-  - Password: `<Your password>`
+## Setting the Docker registry
```
Comment (johnmcollier) [inline docs/_documentations/mdt-che-setupregistries.md]: This section is no longer valid and can be removed

File: docs/_documentations/mdt-che-setupregistries.md [hunk 8]
```diff
@@ -1,25 +1,75 @@
...
+- Artifactory
+    - Address: `<artifactory-hostname>`
+    - Namespace: `<namespace>`
+    - Credentials: Artifactory user name and access token
+- OpenShift Registry
+    - Address: `docker-registry.default.svc:5000`
+    - Namespace: `<project>`
+    - Credentials: OpenShift user name and access token
+
+## Specifying image registries for Codewind
```
Comment (johnmcollier) [inline docs/_documentations/mdt-che-setupregistries.md]: Maybe we should move this section up to be clear about when a registry is needed?

**Comment by sishida** [comment]
Because of merge conflicts, I have made changes on this branch instead: https://github.com/eclipse/codewind-docs/pull/364

**Pull request closed without merging**